clients/acm.d.ts

```diff
@@ -0,0 +1,350 @@
+import {Request} from '../lib/request';
+import {Response} from '../lib/response';
+import {AWSError} from '../lib/error';
+import {Service} from '../lib/service';
+declare class ACM extends Service {
+  /**
```

Comment:
It looks like I'll have to create a custom shape, since only top-level params are actually allowed:
https://github.com/aws/aws-sdk-js/blob/master/lib/service.js#L163